mllib/src/test/scala/org/apache/spark/mllib/recommendation/ALSSuite.scala

```diff
@@ -17,17 +17,16 @@
 
 package org.apache.spark.mllib.recommendation
 
+import org.apache.spark.mllib.recommendation.ALS.BlockStats
+import org.apache.spark.mllib.util.MLlibTestSparkContext
+import org.apache.spark.storage.StorageLevel
+import org.jblas.DoubleMatrix
+import org.scalatest.FunSuite
+
```

Comment:
The imports should be organized in groups: 

https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports